folly/experimental/NestedCommandLineApp.cpp

```diff
@@ -130,8 +130,8 @@ void NestedCommandLineApp::displayHelp(
     // Help for a given command
     auto& p = findCommand(args.front());
     if (p.first != args.front()) {
-      printf("`%1$s' is an alias for `%2$s'; showing help for `%2$s'\n",
-             args.front().c_str(), p.first.c_str());
+      printf("`%s' is an alias for `%s'; showing help for `%s'\n",
```

Comment:
That's just GNU-style quotes (
https://www.gnu.org/prep/standards/html_node/Quote-Characters.html) - an
ill-advised tradition from a bygone era (in my personal opinion), but still
pops up perennially. I'm sure it was the preference of whoever wrote this
code. (An emacs user no doubt, not that there's anything wrong with that
but there seems to be a correlation.)

On Sat, Aug 15, 2015 at 6:13 PM Orvid King <notifications@github.com> wrote:

> In folly/experimental/NestedCommandLineApp.cpp
> <https://github.com/facebook/folly/pull/302#discussion_r37140972>:
>
> > @@ -130,8 +130,8 @@ void NestedCommandLineApp::displayHelp(
> >      // Help for a given command
> >      auto& p = findCommand(args.front());
> >      if (p.first != args.front()) {
> > -      printf("`%1$s' is an alias for `%2$s'; showing help for `%2$s'\n",
> > -             args.front().c_str(), p.first.c_str());
> > +      printf("`%s' is an alias for `%s'; showing help for `%s'\n",
>
> As a side note, what's up with the quotes in this? Why are the opening
> quotes backticks, but the closing quotes just single quotes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/folly/pull/302/files#r37140972>.
>